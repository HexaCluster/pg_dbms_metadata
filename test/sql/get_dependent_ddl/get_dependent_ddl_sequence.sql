SELECT dbms_metadata.get_dependent_ddl('SEQUENCE','table_all','gdmmm');

SELECT dbms_metadata.get_dependent_ddl('SEQUENCE','Sample unlogged table','gdmmm');

-- tests for schema as NULL
SET search_path TO public, gdmmm;

SELECT dbms_metadata.get_dependent_ddl('SEQUENCE','table_all');

SELECT dbms_metadata.get_dependent_ddl('SEQUENCE','Sample unlogged table');
