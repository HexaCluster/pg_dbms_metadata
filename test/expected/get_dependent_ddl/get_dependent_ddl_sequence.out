                                                 get_dependent_ddl                                                 
-------------------------------------------------------------------------------------------------------------------
 CREATE SEQUENCE gdmmm.table_all_a_seq START WITH 1 INCREMENT BY 1 MINVALUE 1 MAXVALUE 2147483647 CACHE 1 NO CYCLE+
                                                                                                                  +
 
(1 row)

                                                        get_dependent_ddl                                                         
----------------------------------------------------------------------------------------------------------------------------------
 CREATE SEQUENCE gdmmm."Sample unlogged table_id_seq" START WITH 1 INCREMENT BY 1 MINVALUE 1 MAXVALUE 2147483647 CACHE 1 NO CYCLE+
                                                                                                                                 +
 
(1 row)

SET
                                                 get_dependent_ddl                                                 
-------------------------------------------------------------------------------------------------------------------
 CREATE SEQUENCE gdmmm.table_all_a_seq START WITH 1 INCREMENT BY 1 MINVALUE 1 MAXVALUE 2147483647 CACHE 1 NO CYCLE+
                                                                                                                  +
 
(1 row)

                                                        get_dependent_ddl                                                         
----------------------------------------------------------------------------------------------------------------------------------
 CREATE SEQUENCE gdmmm."Sample unlogged table_id_seq" START WITH 1 INCREMENT BY 1 MINVALUE 1 MAXVALUE 2147483647 CACHE 1 NO CYCLE+
                                                                                                                                 +
 
(1 row)

